.kitchen.yml

```diff
@@ -12,10 +10,19 @@ platforms:
   - name: chef/centos-6.5
```

Comment:
not sure whether it was intentional, but RHEL platform family support is required for vagrant test.